Client/mods/deathmatch/logic/CClientVectorGraphicDisplay.h

```diff
@@ -34,6 +34,15 @@ class CClientVectorGraphicDisplay final : public CClientDisplay
 
     void Update();
 
+    // Color indexes as per lunasvg::Bitmap
+    static enum ColorIndex
```

Comment:
```suggestion
    enum class eColorIndex
```

Reply:
`enum class` cannot be implicitly converted to int, and the `convert` method from LunaSVG doesn't support it. Even with type specification ie `enum class eColoIndex : int`, this doesn't work. It needs to be a 'classic' enum, so we'll stick with `static enum eColorIndex : int`

Reply:
Does it have to be implicit conversion?

Reply:
it doesn't and looks bad in all cases

Reply:
Debatable